src/models/order.model.js

```diff
@@ -0,0 +1,54 @@
+const { Model } = require('objection');
+const Customer = require('./customer.model');
+const knex = require('../../db/knex');
+const Product = require('./product.model');
+
+Model.knex(knex);
+
+class Order extends Model {
+  static get tableName() {
+    return 'orders';
+  }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: ['currencyCode', 'customer_id', 'paymentStatus'],
+
+      properties: {
+        id: { type: 'integer' },
+        createdAtTimeISO: { type: 'timestamp' },
+        currencyCode: { type: 'string', minLength: 1, maxLength: 255 },
+        customer_id: { type: 'integer' },
```

Comment:
let's use consistent namings please. For db fields and all REST params (both path and query)  lets use **snake_case** and for other cases - **camelCase**